### PR TITLE
intro animation fix + README updates 

### PR DESCRIPTION
**Pull request by lindsaycarr**
intro animation fix (must've just been a small merge conflict or something) as mentioned in #125. Also, added some more documentation based on my experience with building the Gordon animation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ To set up the branch:
 1. Create a new folder and RStudio project just for this storm. Clone the GitHub repository into this project. Because it's a separate folder, you'll be able to add storm-specific data and image files to this folder/project without overwriting files from other storms. We recommend the folder and project name `vizstorm-GIF-[stormname-year]`.
 1. Create and switch to a new storm-specific branch. Use the branch naming convention `Stormname-year`, e.g., `Matthew-2016`.
 1. Edit .gitignore so that the following files are *not* ignored: `*.ind`, `build/status/*.yml`, `viz_config.yml`, `lib/cfg/gd_config.yml`, `2_process/src/filter_sites_custom.R`.
+1. Add a lib/cfg/gd_config.yml. See other storm branches and lib/cfg/gd_config-example.yml for examples.
```
Comment (aappling-usgs) [inline README.md]: i think this is in line 21 (a few steps below) and belongs after the creation of the Drive folder. But I like your language here - feel free to replace/edit my language below to make it more concise.

File: README.md [hunk 2]
```diff
@@ -22,13 +23,14 @@ To set up the branch:
 
 To build the storm GIF:
 1. Make your best guess about an appropriate `bbox` in viz_config.yml.
-1. Make your best guess about a good site list in 2_process/src/filter_sites_custom.R (this can be broad, at the cost of needing to pull and plot more NWIS data in early iterations).
+1. Make your best guess about a good site list in 2_process/src/filter_sites_custom.R (this can be broad, at the cost of needing to pull and plot more NWIS data in early iterations). It can be helpful to see the site numbers on the figures with the spark lines as you iterate through site selection. One way to do this is to add this `text(full_poly$x[1]-strwidth("5555"), full_poly$y[1], labels=site)` to the `prep_spark_line_fun` where the stage shapes are being added to the plot. Just delete when you're done iterating.
```
Comment (aappling-usgs) [inline README.md]: ahh, good quick fix!

File: README.md [hunk 3]
```diff
@@ -22,13 +23,14 @@ To set up the branch:
...
 1. Iterate through choices of bbox, site list, ocean and city names, etc. until you have the gif_frames (pngs) looking the way you want.
 1. Build the animated GIF with `scmake('6_visualize/out/animation_a.gif')`. It often takes a long time to build all the pngs, so if you want to preview an animated GIF with only some frames, consider specifying a subset of frames with the `task_names` argument to `combine_animation_frames()`.
+1. Once you're happy with the way the animated GIF looks, you will need to build the intro frames. Do this by making sure `task_names=NULL` for the target `6_visualize/log/6_intro_gif_tasks.ind` in `6_visualize.yml`. Run `scmake('6_visualize/log/6_intro_gif_tasks.ind')` to build the intro frames. Then, run `scmake(6_visualize/out/animation_a.gif')` to incorporate the intro frames into the full animated GIF (note that you might need to add `force = TRUE` if the file already exists).
```
Comment (aappling-usgs) [inline README.md]: maybe add that you might need force=TRUE not just if animation_a.gif already exists but also if '6_visualize/log/6_intro_gif_tasks.ind' or '6_visualize/log/6_storm_gif_tasks.ind' already exist but the pngs still need [re]building?

Comment (lindsaycarr) [reply README.md]: oh right, good point

File: README.md (hunk 1 above)
Comment (lindsaycarr) [reply README.md]: oh I guess I thought those were different things. It was just never said that `gd_config-example.yml` needed to be changed or copied to be `gd_config.yml` before it started talking about using `gd_config.yml`. I'll just clarify that in the existing step

Comment (lindsaycarr) [reply README.md]: Oh nope, I get it now. Running `scipiper::gd_config(config_file="lib/cfg/gd_config.yml", folder="drivefolderid")` does actually create that file. 👍

**Pull request merged**